luaui/Widgets/gui_comselect.lua

```diff
@@ -0,0 +1,128 @@
+function widget:GetInfo()
+	return {
+		name = "Commander Selector",
+		desc = "Allows for proper keybind selection of any flavour of commanders",
+		author = "Hornet, with some ZK raiding",
+		date = "03 June 2024",
+		license = "GNU GPL, v2 or later",
+		layer = 1001,
```

Comment:
Stick to zero
```suggestion
		layer = 0,
```
unless there's a good reason not to, in which case document it.
```suggestion
		layer = 1001, -- after X, before Y
```

Reply:
Yes but
1) in ZK the commander selector part of a larger widget that draws a lot of GUI, and as far as I can tell the widget's layer is set with drawing order in mind
2) layer only matters by creating an ordering relationship with other widgets and BAR just has a different set of widgets than ZK